### PR TITLE
add root CA bundle and use it for making https request in pilot

### PR DESCRIPTION
**Pull request by quanjielin**
This PR includes 
1. add public CA bundle to pilot container(https://github.com/istio/istio/issues/1419)
2. use the CA bundle to support https call to fetch jwt key(https://github.com/istio/istio/issues/4917), this is the fix for workaround to skip verification(made in https://github.com/istio/istio/pull/5238)

### CODE REVIEW

File: pilot/pkg/model/jwks_resolver.go [hunk 1]
```diff
@@ -102,17 +109,30 @@ func newJwksResolver(expireDuration, evictionDuration, refreshInterval time.Dura
...
-		client: &http.Client{
+		httpClient: &http.Client{
 			Timeout: jwksHTTPTimeOutInSec * time.Second,
-
-			// TODO: pilot needs to include a collection of root CAs to make external
-			// https web request(https://github.com/istio/istio/issues/1419).
 			Transport: &http.Transport{
-				TLSClientConfig: &tls.Config{InsecureSkipVerify: true},
+				TLSClientConfig: &tls.Config{
+					InsecureSkipVerify: true,
```
Comment (kyessenov) [inline pilot/pkg/model/jwks_resolver.go]: remove this flag to validate.

**Comment by quanjielin** [comment]
@kyessenov I tried to create a folder and put bundle there but seems didn't work; guess it's because pilot container is based on “scratch” so we have no bash in the container, RUN mkdir -p doesn't work.

File: pilot/pkg/model/jwks_resolver.go [hunk 2]
```diff
@@ -78,7 +84,8 @@ type jwksResolver struct {
 	// map key is jwksURI, map value is jwtPubKeyEntry.
 	keyEntries sync.Map
 
-	client        *http.Client
+	httpsClient   *http.Client
```
Comment (diemtvu) [inline pilot/pkg/model/jwks_resolver.go]: [nit] use httpClient and secureHttpClient or something like that. httpClient and httpsClient is too much similar and may cause bug due to mistyping.

Comment (quanjielin) [reply pilot/pkg/model/jwks_resolver.go]: done

**Pull request merged**